assets/admin/editor-wizard/steps/course-upgrade-step.js

```diff
@@ -10,6 +10,7 @@ import { useSelect } from '@wordpress/data';
  */
 import { EXTENSIONS_STORE } from '../../../extensions/store';
 import senseiProUpsellImage from '../../../images/sensei-pro-upsell.png';
+import CheckIcon from '../../../icons/checked.svg';
```

Comment:
This comes from a change I had from `feature/course-lesson-wizard` that @renatho's parent branch don't have yet. Please ignore this.